### DISCUSSION TITLE
What do bots do when no players are online?

### DISCUSSION THREAD

**Discussion by christiansenandi** [celguar/mangosbot-bots#79] [Q&A]
Hi,

First, thanks for this and the wotlk pack.. really enjoying it so far!

I been wondering about this question... Does the bots level up when server is online with no players?

**Comment by celguar** [comment]
@christiansenandi  Hello!
With default settings only 10% of random bots are active at a time. Others are mostly idle, doing nothing. 10% are cycled, so all bots get their time to be active.
They usually try to do quests, or grind mobs, or go sell junk to vendor. But they are not as fast to level up as humans.
Also with default settings bots get random level from 1 to max level (gradually while server is running)
